docs/portal/team-members.md

```diff
@@ -56,7 +60,7 @@ To re-send the invitation for a team member, complete the following steps:
 
 ## Edit Information for a Team Member
 
-You can update any of the values: First Name, Last Name, Email, or Job Function.
+Admins can update any of the values: First Name, Last Name, Email, Job Function, or Role.
```

Comment:
nit: "Admin users" suggestion for here too

Reply:
yep good call. done.